mappings/net/minecraft/entity/ai/brain/task/SometimesTask.mapping

```diff
@@ -0,0 +1,15 @@
+CLASS net/minecraft/class_4821 net/minecraft/entity/ai/brain/task/SometimesTask
```

Comment:
Not sure about this name?

Reply:
Most likely `TimeLimitedTask` then, halting a parent task based on an arbitrary time limit in addition. similar to `ConditionalTask` (class_4820), which also adds an extra condition on task running.

Reply:
Renamed to `TimeLimitedTask`